src/odin/generate/data_dictionary/duck_db.py

```diff
@@ -12,10 +13,11 @@
 from odin.utils.locations import CUBIC_QLIK_DATA
 from odin.utils.locations import AFC_DATA
 from odin.utils.locations import CUBIC_ODS_REPORTS
+from odin.utils.parquet import ds_from_path
 import odin.generate.data_dictionary.cubic_reports_sql as cubic_sql
 
 
-DB_FILE = "fares_data_repository.db"
+DB_FILE = f"fares_data_repository_{AWS_ENV}.db"
```

Comment:
Yo. Do you mind adding this prefix only for DEV or non-PROD? It's a little cleaner.